src/v2/cloud_tasks_client.ts

```diff
@@ -2052,6 +1992,66 @@ export class CloudTasksClient {
     return this._pathTemplates.taskPathTemplate.match(taskName).task;
   }
 
+  /**
```

Comment:
@alexander-fenster @xiaozhenliu-gg5 if I'm reading this right, it looks like the order of the generated methods are bouncing a round a bit.  They don't seem to be in alphabetical order, so I'm not sure if this is intended or not.  Do we have jitter here?

Reply:
We had a sequence of logic changes here trying to make some weird APIs work. We do sort them, but there appears to be a bug there :)

```
  augmentedService.pathTemplates = Object.values(uniqueResources).sort();
```
here `sort()` is applied to objects and the results are quite random, as you see.

Let me fix this and I'll regen the APIs today with a set of mass PRs. I'll `repo reject` the open ones.

Reply:
https://github.com/googleapis/gapic-generator-typescript/pull/231